x-pack/plugins/apm/server/lib/transactions/get_latency_charts/index.ts

```diff
@@ -80,16 +85,7 @@ async function searchLatency({
             min_doc_count: 0,
             extended_bounds: { min: start, max: end },
           },
-          aggs: {
-            avg: { avg: { field } },
-            pct: {
-              percentiles: {
-                field,
-                percents: [95, 99],
-                hdr: { number_of_significant_value_digits: 2 },
-              },
-            },
-          },
+          aggs: getLatencyAggregation(latencyAggregationType, field),
```

Comment:
Can we rename `field`?

```suggestion
          aggs: getLatencyAggregation(latencyAggregationType, transactionDurationField),
```